profile-change/common/pom.xml

```diff
@@ -13,31 +13,37 @@
         <dependency>
             <groupId>dk.dbc</groupId>
             <artifactId>vipcore</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>2.0-SNAPSHOT</version>
             <classifier>vipcore-client</classifier>
         </dependency>
 
         <dependency>
             <groupId>dk.dbc</groupId>
             <artifactId>openagency</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>2.0-SNAPSHOT</version>
             <classifier>vipcore-httpclient</classifier>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+
```

Comment:
Is that necessary?

Reply:
Yes, without it results in a java.lang.NoSuchMethodError: 'boolean org.glassfish.jersey.internal.config.SystemPropertiesConfigurationModel.hasProperty(java.lang.String)'
        at dk.dbc.scan.common.ProfileServiceActionsIT.testGetProfiles(ProfileServiceActionsIT.java:59)